app/assets/javascripts/admin/addon/components/themes-list.js

```diff
@@ -107,4 +141,40 @@ export default class ThemesList extends Component {
   navigateToTheme(theme) {
     this.router.transitionTo("adminCustomizeThemes.show", theme);
   }
+
+  @action
+  toggleAllInactive() {
+    const markedToDelete = this.selectedCount === 0;
+    this.inactiveThemes.forEach((theme) =>
+      theme.set("markedToDelete", markedToDelete)
+    );
+  }
+
+  @action
+  deleteConfirmation() {
+    this.modal.show(DeleteThemesConfirm, {
+      model: {
+        themes: this.selectedThemesOrComponents,
```

Comment:
nit, I think this key should be `selectedThemesOrComponents` too so it is clear in the modal it could be either